### PR TITLE
feat(web app analytics): InPlayer user id

### PR DESCRIPTION
**Pull request by CarinaDraganJW**
## Description

- Use `id` field from the response from the JWP /  InPlayer user and use it to populate the `oaid` field with the analytics pings

### Steps completed:

According to our definition of done, I have completed the following steps:

- [ ] Acceptance criteria met
- [ ] Unit tests added
- [ ] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [ ] Rebased & ready to merge without conflicts
- [ ] Reviewed own code

### CODE REVIEW

File: src/hooks/useOttAnalytics.ts [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
+import jwtDecode from 'jwt-decode';
 
 import type { PlaylistItem } from '#types/playlist';
 import { useConfigStore } from '#src/stores/ConfigStore';
+import { useAccountStore } from '#src/stores/AccountStore';
+import type { DecodedJwtToken } from '#types/account';
 
 const useOttAnalytics = (item?: PlaylistItem, feedId: string = '') => {
   const analyticsToken = useConfigStore((s) => s.config.analyticsToken);
+  const auth = useAccountStore((state) => state.auth);
```
Comment (budzworthy) [inline src/hooks/useOttAnalytics.ts]: So, I think we can actually get the user ID from the inplayer sdk response to the login method, which is stored in accountStore.user.id (I think).

**Review by budzworthy — COMMENTED** [review]
I think we can avoid having to decode the token. Any luck with adding the user id to the player analytics?

File: src/hooks/useOttAnalytics.ts [hunk 2]
```diff
@@ -2,9 +2,15 @@ import { useCallback, useEffect, useState } from 'react';
...
 import type { PlaylistItem } from '#types/playlist';
 import { useConfigStore } from '#src/stores/ConfigStore';
+import { useAccountStore } from '#src/stores/AccountStore';
 
 const useOttAnalytics = (item?: PlaylistItem, feedId: string = '') => {
   const analyticsToken = useConfigStore((s) => s.config.analyticsToken);
+  const user = useAccountStore((state) => state.user);
+
+  // oaid (Ott App User ID) for analytics
+  const oaid: number | undefined = Number(user?.id);
```
Comment (AntonLantukh) [inline src/hooks/useOttAnalytics.ts]: Should we only send it for JW subscription model?

Comment (CarinaDraganJW) [reply src/hooks/useOttAnalytics.ts]: @dbudzins Do we need to check here the subscription model as well?

Comment (budzworthy) [reply src/hooks/useOttAnalytics.ts]: You mean for JW instead of cleeng?

Comment (AntonLantukh) [reply src/hooks/useOttAnalytics.ts]: Looks like Number(undefined) will give us a different result.

Comment (CarinaDraganJW) [reply src/hooks/useOttAnalytics.ts]: I added a check for it 🙂

**Review by AntonLantukh — APPROVED** [review]

**Pull request merged**